### PR TITLE
Tecnologia

### PR DESCRIPTION
**Pull request by rodrigoWolox**
## Summary
- Tecnology card  first commit

## Screenshot
image

## Trello Card
https://trello.com/c/FpkyXJoi/22-tecnolog%C3%ADa

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,21 @@
   <link rel="stylesheet" href="application.css" type="text/css">
 </head>
 <body>
-  <h1 class="page-title">Front-End Tools</h1>
+  <div class="container">
```
Comment (mmamaniwolox) [inline index.html]: You can add this class to body element and delete this div

File: index.html [hunk 2]
```diff
@@ -10,6 +10,21 @@
...
+    <h1 class="text-big page-title">Front-End Tools</h1>
+    <div class="tech-section">
```
Comment (mmamaniwolox) [inline index.html]: This div is for this technology or for every technology in front end tools?

File: index.html [hunk 3]
```diff
@@ -10,6 +10,21 @@
...
+      <div class="tech-container">
+        <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS logo">
+        <p class="tech-title">ReactJS</p>
+        <p class="tech-sub">Release: 2013</p>
+        <p class="tech-desc">
```
Comment (mmamaniwolox) [inline index.html]: Put all the text in just one line. 
```
<p class="tech-desc">
  Text here
</p>
```

File: scss/fonts.scss [hunk 4]
```diff
@@ -1,3 +1,7 @@
+* {
```
Comment (mmamaniwolox) [inline scss/fonts.scss]: Remember this should be in `styles.scss` file

File: scss/styles.scss [hunk 5]
```diff
@@ -1,3 +1,3 @@
-* {
-  font-family: 'Roboto', sans-serif;
-}
+.container {
+  padding: 150px;
+}
```
Comment (mmamaniwolox) [inline scss/styles.scss]: Add a tailing line

File: scss/tech.scss [hunk 6]
```diff
@@ -1,3 +1,37 @@
...
+		position: absolute;
+		top: 15px;
+		width: 100%;
+		z-index: -1;
+  }
+}
+
 .tech-logo {
   height: 100px;
+  width: 100px;
```
Comment (mmamaniwolox) [inline scss/tech.scss:31]: Try without this width, height should be enought

File: scss/tech.scss [hunk 7]
```diff
@@ -1,3 +1,37 @@
+@import 'colors';
+@import 'fonts';
+
+.tech-container {
```
Comment (mmamaniwolox) [inline scss/tech.scss:4]: This is the background border right? This shouldnt be the main class, the tech container should be the one with green backgound and then use a pseudo-element for the external border

File: scss/tech.scss [hunk 8]
```diff
@@ -1,3 +1,37 @@
...
+  border: 1px solid;
+  border-color: $earls-green;
+  border-radius: 5px;
+  display: inline-block;
+  height: auto;
+  margin: 20px;
+  padding: 40px;
+  position: relative;
+  text-align: center;
+  width: 20%;
```
Comment (mmamaniwolox) [inline scss/tech.scss:14]: Use a fixed width here

File: scss/tech.scss [hunk 9]
```diff
@@ -1,3 +1,37 @@
...
+
+  &::after {
+    background-color: $earls-green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    left: 15px;
+		position: absolute;
```
Comment (mmamaniwolox) [inline scss/tech.scss:22]: What happened here with the indentation? :O

File: scss/titles.scss [hunk 10]
```diff
@@ -1,17 +1,37 @@
...
     height: 90px;
-    right: 250px;
     position: absolute;
+    right: 150px;
     width: 70px;
     z-index: -1;
   } 
 }
+
+.tech-title {
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This should be in tech.scss

File: scss/titles.scss [hunk 11]
```diff
@@ -1,17 +1,37 @@
...
+  margin: 10px;
+}
+
+.tech-sub {
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This should be in tech.scss

File: scss/titles.scss [hunk 12]
```diff
@@ -1,17 +1,37 @@
...
+  font-weight: 900;
+  margin: 20px;
+}
+
+.tech-desc {
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This should be in tech.scss

File: scss/titles.scss [hunk 13]
```diff
@@ -1,17 +1,37 @@
...
+  margin: 30px 15px 25px;
+}
+
+.tech-title,
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This should be in tech.scss

File: scss/titles.scss [hunk 14]
```diff
@@ -1,17 +1,37 @@
...
+.tech-sub,
+.tech-desc {
+  display: block;
+  font-size: $text-medium;
+}
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Add a tailing line

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (rodrigoWolox) [reply index.html]: the div tech-section is to put all tech-container (one for each tech)

File: scss/fonts.scss (hunk 4 above)
Comment (rodrigoWolox) [reply scss/fonts.scss]: Don't know how that get there T_T

**Pull request closed without merging**